microbit/src/10-punch-o-meter/the-challenge.md

```diff
@@ -1,27 +1,47 @@
-# The challenge
+<!-- # The challenge -->
 
-To keep things simple, we'll measure the acceleration only in the X axis while the board remains
+# 課題
+
+<!-- To keep things simple, we'll measure the acceleration only in the X axis while the board remains
 horizontal. That way we won't have to deal with subtracting that *fictitious* `1g` we observed
 before which would be hard because that `1g` could have X Y Z components depending on how the board
-is oriented.
+is oriented. -->
+
+話を簡単にするために、ここではボードを水平に保った状態でX軸の加速度だけを測ることにします。
```

Comment:
> That way we won't have to deal with subtracting that *fictitious* `1g` we observed
before which would be hard because that `1g` could have X Y Z components depending on how the board
is oriented.

この部分は翻訳漏れでしょうか？(もしあえてだったらすみません

Reply:
本当ですね。なぜか抜けていました。その部分を追加したものをプッシュしました。

Reply:
ありがとうございます！